inspire/modules/workflows/static/js/workflows/actions/core_approval.js

```diff
@@ -120,6 +138,18 @@ define(
         this.on("rejectConfirmed", this.doRejection)
         console.log("Core approval init");
       });
+
+      $(function(){
```

Comment:
Will it work to put this initialization inside the `this.after("initialize"` part of this component? Without the `$function()` part as well?